headers/xdp/parsing_helpers.h

```diff
@@ -101,6 +102,39 @@ static __always_inline int parse_ethhdr(struct hdr_cursor *nh, void *data_end,
 	return h_proto; /* network-byte-order */
 }
 
+static __always_inline int skip_ip6hdrext(struct hdr_cursor *nh,
+					  void *data_end,
+					  __u8 currhdr)
+{
+	for (int i = 0; i < 8; ++i) {
```

Comment:
The number of loop iterations should be a define, like VLAN_MAX_DEPTH above

Reply:
Also, you can probably lower it to 6, since the code only handles 6 different headers, and none of them should occur more than once (according to the RFC).